corpus/expressions.txt

```diff
@@ -428,8 +428,15 @@ Instance expressions
 
 class C {
   def main() {
-    a = new B
-    c = new D(e, f)
+    val a = new B
+    val c = new D(e, f)
+    val e = new E:
```

Comment:
For posterity, can you also add an example of the quiet syntax `new:`

```scala
scala> trait Hello:
     |   def x: Int
     |
// defined trait Hello

scala> val x: Hello = new:
     |   def x = 25
     |
val x: Hello = anon$1@6bd92538
```

Reply:
Done - https://github.com/tree-sitter/tree-sitter-scala/pull/131/commits/14a22e7baa5f26c810d0ff5865fa2fa83f24b0c3